NOTICE

```diff
@@ -1,15 +1,13 @@
-/**
- * Copyright 2021 Bonitasoft S.A.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+Copyright 2021-present Bonitasoft S.A.
```

Comment:
If we remove the TSDoc format, will the notice be in TSDoc format in the TS files?

Reply:
If you are thinking about the eslint notice plugin, there is no issue (you can recheck it).
The plugin is not using the NOTICE file but `config/license-header.js (see `.eslintrc.js`)

Reply:
I didn't remember that it was not linked.

If we do that, we should remove `NOTICE` from the `scripts` of `package.json` and from `lint-staged`:
```
    "lint": "eslint \"*/**/*.{js,mjs,ts}\" NOTICE --max-warnings 0 --quiet --fix",
    "lint-check": "eslint \"*/**/*.{js,mjs,ts}\" NOTICE --max-warnings 0",
```

Reply:
The NOTICE directive in the npm script was introduced in #142. I don't understand why we want to run `eslint` on the NOTICE file.
I guess this is because previously set this configuration in bpmn-visualization: https://github.com/process-analytics/bpmn-visualization-js/pull/129

So to me, we should drop the directive from the npm script as you mentioned.

Reply:
I didn't remember exactly. I used an article to set this configuration, but I can't find it :(

Reply:
I have removed the NOTICE file from the eslint processing, see dd7a129